CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - Since handles are no longer globally unique, you must not pass handles generated by one ASC to a different ASC.
     - `FGameplayAbilitySpecHandle` instances are now invalid handles when constructed directly. The only way to get a valid handle is from giving an ability to an ASC.
     - Granting an ability from an `FGameplayAbilitySpecDef` must now be done through the `FGameplayAbilitySpec::GiveAbilityFromSpecDef`/`GiveAbilityAndActivateOnceFromSpecDef` functions.
+- Removed `USpatialStaticComponentView`; similar functionality is now provided in `ViewCoordinator`.
```

Comment:
Does this need a release note - this is entirely internal

Reply:
Discussed offline - it's a symbol we do export so this is external in some sense.